### PR TITLE
ISPN-13673 operator deployment diagrams

### PR DESCRIPTION
**Pull request by oraNod**
https://issues.redhat.com/browse/ISPN-13673

### CODE REVIEW

File: documentation/asciidoc/topics/con_operator_deployments.adoc [hunk 1]
```diff
@@ -0,0 +1,22 @@
+//This topic is intended as downstream content.
+[id='infinispan-operator-deployments_{context}']
+= {ispn_operator} deployments
+
+When you install {ispn_operator}, it extends the Kubernetes API with Custom Resources (CR) for deploying and managing {brandname} clusters on {openshift}.
```
Comment (ryanemerson) [inline documentation/asciidoc/topics/con_operator_deployments.adoc:5]: CRs and CRDs are backwards here. CRDs extend the k8s API, CRs are what the user creates/applies

File: documentation/asciidoc/topics/con_operator_deployments.adoc [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+To interact with {ispn_operator}, {openshiftshort} users apply Custom Resource Definitions (CRDs) through the {osweb} or `oc` client.
+{ispn_operator} listens for `Infinispan` CRs and automatically provisions native resources, such as StatefulSets and Secrets, in addition to any resources for your {brandname} deployment.
```
Comment (ryanemerson) [inline documentation/asciidoc/topics/con_operator_deployments.adoc:8]: Maybe we should clarify what is meant by "any resources" here. I assume you're referring to the downloading of db drivers, user libraries etc

File: documentation/asciidoc/topics/con_operator_deployments.adoc [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+.Custom resources
+image::custom-resources.png[This illustration depicts how an {openshiftshort} user can pass custom resources to {ispn_operator}.]
+
+A single {ispn_operator} installation can manage multiple {brandname} clusters in separate namespaces.
+Each time an {openshiftshort} user applies CRDs to modify the deployment, {ispn_operator} applies the changes globally to all {brandname} clusters.
```
Comment (ryanemerson) [inline documentation/asciidoc/topics/con_operator_deployments.adoc:14]: applies CRs

**Review by ryanemerson — CHANGES_REQUESTED** [review]

File: documentation/asciidoc/topics/con_operator_deployments.adoc (hunk 1 above)
Comment (oraNod) [reply documentation/asciidoc/topics/con_operator_deployments.adoc:5]: facepalm.gif fixed, thanks.

File: documentation/asciidoc/topics/con_operator_deployments.adoc (hunk 2 above)
Comment (oraNod) [reply documentation/asciidoc/topics/con_operator_deployments.adoc]: I've reworded this to say that the operator also configures Infinispan services according to user CR specifications. That's what I was driving at before. The operator sets up all the k8s native objects in addition to any Infinispan cluster configuration.

**Review by pruivo — APPROVED** [review]

**Pull request merged**